### PR TITLE
Issue/confsrvdev 23885 fixing xss in 2 0 x

### PR DESCRIPTION
**Pull request by ajaysharma2**
confsrvdev 23885 Fixed xss
changing java 5 to java8

### CODE REVIEW

File: pom.xml
Comment (rjatkins) [inline pom.xml:1]: We also need to update the distributionManagement to point to
```
    <distributionManagement>
        <repository>
            <id>atlassian-3rdparty</id>
            <url>https://packages.atlassian.com/maven/3rdparty</url>
        </repository>
    </distributionManagement>
```

File: src/main/java/org/apache/velocity/tools/view/VelocityViewServlet.java [hunk 1]
```diff
@@ -353,7 +353,7 @@ protected void error(HttpServletRequest request,
             html.append("<head><title>Error</title></head>\n");
             html.append("<body>\n");
             html.append("<h2>VelocityView : Error processing a template for path '");
-            html.append(path);
+            html.append(StringEscapeUtils.escapeHtml(ServletUtils.getPath(request)));
```
Comment (rjatkins) [inline src/main/java/org/apache/velocity/tools/view/VelocityViewServlet.java:357]: Require `ATLASSIAN:` comment before this, explaining what we changed

File: src/test/java/org/apache/velocity/tools/test/blackbox/ViewToolsTests.java
Comment (rjatkins) [inline src/test/java/org/apache/velocity/tools/test/blackbox/ViewToolsTests.java:1]: Why was this test deleted?

**Review by rjatkins — CHANGES_REQUESTED** [review]

File: src/test/java/org/apache/velocity/tools/test/blackbox/ViewToolsTests.java
Comment (ajaysharma2) [reply src/test/java/org/apache/velocity/tools/test/blackbox/ViewToolsTests.java:1]: In upstream also it is deleted, Also it needs sample project deployment for testing it
I am looking into it if can be restored and tested

File: pom.xml
Comment (ajaysharma2) [reply pom.xml:1]: Done

**Review by rjatkins — APPROVED** [review]
Looks good!

**Pull request merged**